export-to-csv.ts

```diff
@@ -153,12 +153,19 @@ export class ExportToCsv {
      * Create Body
      */
     private _getBody() {
-        const keys = Object.keys(this._data[0]);
```

Comment:
In that library the first row of passed CSV data says how many columns can have rows, I have changed that behaviour which prevents us to hack the data